aries-site/src/pages/templates/banner-notifications.mdx

```diff
@@ -1,3 +1,135 @@
+- Task related alerts
+- To show if task is successful. Instead, use [Toast Notifications](/templates/toast-notifications).
+
+## Guidelines
+
+### Format
+
+- Place below global header.
+- Show one banner at a time.
+- Implement inline with the page content to ensure no other content is obscured.
+- Limit to 192 characters to avoid truncation. Content will be restricted to a single line. If there is insufficient width for the whole message, it will be truncated.
+
+### Things to Avoid
+
+- Do not allow dismissal of the banner (`onClose`) when information is critical (such as potential access loss).
+- Do not stack banners on top of one another; use a single banner at a time.
+
+### Behavior
+
+- Never sticky, should scroll when there is other content
+- When clicked, should direct user to notification center when clicked.
+- When dismissed, should remove the banner from the screen but the notification will still exist in the notification center.
+
+### Placement
+
+- Banner notifications should always be located directly below the Global Header, above the Page Header.
+
+## Variants
+
+### Informational
+
+Used when there is a:
+
+- New feature
+- Upcoming release
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe#pairProps"
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListReadMoreExample.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListReadMoreExample.js"
+  height={{ min: 'small' }}
+>
+  <BannerNotificationInfo />
+</Example>
+
+### Critical
+
+Used to warn the user about:
+
+- Unavailable access to the platform due to the system
+- Scheduled maintenances (good for persistent)
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe#pairProps"
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListReadMoreExample.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListReadMoreExample.js"
+  height={{ min: 'small' }}
+>
+  <BannerNotificationCritical />
+</Example>
+
+### Warning
+
+Used to warn the user about:
+
+- Subscriptions about to expire (can be dismissed)
+- Session timeout
+
+<Example
+  docs="https://v2.grommet.io/namevaluelist?theme=hpe#pairProps"
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListReadMoreExample.js",
+  ]}
+  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/namevaluelist/Examples/NameValueListReadMoreExample.js"
+  height={{ min: 'small' }}
+>
+  <BannerNotificationWarning />
+</Example>
+
+## Do’s and Don’ts
+
+<BestPracticeGroup>
```

Comment:
Think the Do and Don't should stack one on top of each other so give the banner a more realistic width.